src/Store/Reducers/favoriteReducer.js

```diff
@@ -1,29 +1,32 @@
 // Store/Reducers/favoriteReducer.js
 
-const initialState = { favoritesDrill: [] }
+const initialState = { favoritesDrill: [] };
 
 function toggleFavorite(state = initialState, action) {
-  let nextState
+  let nextState;
+
+  console.log(state);
```

Comment:
Once you finish your test, you can delete these two logs (or set a better to comment to understand them)
But I think you can delete them